core/src/main/java/hudson/model/AbstractCIBase.java

```diff
@@ -93,13 +93,26 @@ protected void killComputer(Computer c) {
     * Package-protected, but accessed API
     * ============================================================================================================== */
 
-    /*package*/ final CopyOnWriteArraySet<String> disabledAdministrativeMonitors = new CopyOnWriteArraySet<>();
+    private Set<String> disabledAdministrativeMonitors = new HashSet<>();
```

Comment:
There is a gigantic comment above that you might want to consider removing

Reply:
It was probably already obsolete after https://github.com/jenkinsci/jenkins/pull/3873